### PR TITLE
Add ISBN to CITATION.cff

### PR DESCRIPTION
**Pull request by rikhuijzer**
The only place that I could add the ISBN was in the preferred citation area (https://github.com/citation-file-format/citation-file-format/blob/main/examples/1.2.0/pass/key-complete/CITATION.cff), so that's why I added that.

I have no idea what the precise effects are on Zotero but I guess it is an improvement.

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -1,27 +1,47 @@
...
-  -
-    type: url
+  - type: url
     value: "https://juliadatascience.io/"
+
+keywords:
+  - "Julia language"
+  - "programming"
+  - "software"
+  - "graphics"
```
Comment (storopoli) [inline CITATION.cff:27]:
```suggestion
  - "data manipulation"
  - "data visualization"
```

File: CITATION.cff [hunk 2]
```diff
@@ -1,27 +1,47 @@
...
 license: "CC-BY-NC-SA-4.0"
-message: "If you use this software, please cite it using these metadata."
+
 repository-code: "https://github.com/JuliaDataScience/JuliaDataScience"
+
 title: "Julia Data Science"
+
 version: 1.0.0
+
+preferred-citation:
```
Comment (storopoli) [inline CITATION.cff:37]: Don't we have to put `authors` here also in the `preferred-citation`?

**Review by storopoli — APPROVED** [review]

File: CITATION.cff (hunk 2 above)
Comment (rikhuijzer) [reply CITATION.cff:37]: I have no idea, I was hoping that systems would pull it from the layer up if the info is missing. If we merge, then I can check what Zotero makes from it

Comment (storopoli) [reply CITATION.cff:38]: Ok! Let's try that.

**Pull request merged**